packages/website/ts/types.ts

```diff
@@ -764,4 +764,16 @@ export interface Package {
     description: string;
     link: ALink;
 }
+
+export type VoteOutcome = 'accepted' | 'rejected';
+export type VoteTime = 'upcoming' | 'happening';
+export type VoteStatus = VoteOutcome | VoteTime;
+export type ZeipId = 39 | 24 | 23 | 25;
```

Comment:
This might be annoying to update, what are your thoughts on having this over just number?